packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -519,6 +551,29 @@ describe('src/cy/commands/querying', () => {
         })
       })
 
+      it('can get alias with logging off', { protocolEnabled: true }, () => {
+        const logs = []
+        let hiddenLog
+
+        cy.on('log:added', (attrs, log) => {
+          logs.push(log)
+        })
+
+        cy.on('_log:added', (attrs, log) => {
+          hiddenLog = log
+        })
+
+        cy.get('body', { log: false }).as('b').get('@b', { log: false })
```

Comment:
Does the first `get` need to have `log: false` set?

```suggestion
        cy.get('body').as('b').get('@b', { log: false })
```

Reply:
[`ccbeea0` (#28531)](https://github.com/cypress-io/cypress/pull/28531/commits/ccbeea0bfaf24861ebb0d38c53ee6cba94bbddd2)